python/tvm/relay/frontend/tensorflow.py

```diff
@@ -294,11 +294,30 @@ def _impl(inputs, attr, params):
 
             attr['padding'] = [0, 0]
 
+        elif attr['padding'] == 'EXPLICIT':
+            warnings.warn(
+                'Explicit padding is for now only supporting same padding '
+                'before/after. It uses values 0 and 2, out of its 4 values.'
+            )
+            if 'explicit_paddings' not in attr:
+                raise tvm.error.OpAttributeInvalid(
+                    'EXPLICIT padding mode requires the'
+                    ' `explicit_padding` attribute.'
+                )
+            h = attr['explicit_paddings'][0]
+            v = attr['explicit_paddings'][2]
+            attr['padding'] = [h, v]
+
         else:
             msg = 'Value {} in attribute "padding" of operator Conv is not ' \
                   'valid.'
             raise tvm.error.OpAttributeInvalid(msg.format(attr['padding']))
 
+        # Not needed in the Relay API for `conv`.
+        # Consumed only if `padding` was set to `EXPLICIT`.
+        if opname == 'conv' and 'explicit_paddings' in attr:
+            del attr['explicit_paddings']
```

Comment:
Typically unused attributes are added to the ignores option of AttrCvt. It probably makes more sense to do that with explicit_padding rather than del this way.

Reply:
+1

Reply:
In this commit `explicit_paddings` is used a few lines above. If the padding is set to `EXPLICIT`, TF expects `explicit_paddings` to be set, or rejects the call as invalid. In Relay, I have translated the `EXPLICIT` mode to set the 2 explicit padding values (this mode in TF can set the 4 padding values).

On this one I have decided to proceed with `del` after "using" the value, so the new `EXPLICIT` mode is (partially) supported, and execution can proceed.

Adding `explicit_paddings` to the ignore list was an option (I use it in this PR for the `half_pixel_centers` new attribute), but I saw an opportunity for partial and sane support of `EXPLICIT`. Please let me know what you prefer, and this can be adapted in no time.

Reply:
then adding into `ignores` is more consistent with rest of codes

Reply:
I don't understand the "then", but why not for now.